src/connectors/hits/connectHits.ts

```diff
@@ -115,6 +115,8 @@ const connectHits: HitsConnector = function connectHits(
         if (!results) {
           return {
             hits: [],
+            sendEvent,
```

Comment:
How come this isn't covered by a test? That's public API.

Reply:
I guess it wasn't tested because at `init` phase, we cannot send events.

Reply:
It's still important to test it because **users bind events during `init`**. If it's not passed, the API is not usable because the reference to `sendEvent` doesn't exist.

Reply:
Yeah right. It makes sense.

Reply:
https://github.com/algolia/instantsearch.js/pull/4570/commits/4bff938aa7144695c1c90ec86e3c3990cad3c8a5